283.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {number[]} nums
+ * @return {void} Do not return anything, modify nums in-place instead.
+ */
+var moveZeroes = function (nums) {
+  let j = 0;
+  let i = 1;
```

Comment:
```suggestion
  let i = 0;
  let j = 1;
```
[nitpick] Multiple variables for index are commonly in alphabetical order. So it would cause less confusion to readers 👍